packages/wdio-logger/src/node.ts

```diff
@@ -167,9 +168,9 @@ getLogger.setLogLevelsConfig = (logLevels = {}, wdioLogLevel = DEFAULT_LEVEL) =>
         /**
          * either apply log level from logLevels object or use global logLevel
          */
-        const logLevel = typeof logLevelsConfig[logLevelName] !== 'undefined' ? logLevelsConfig[logLevelName] : process.env.WDIO_LOG_LEVEL
+        const logLevel = typeof logLevelsConfig[logLevelName] !== 'undefined' ? logLevelsConfig[logLevelName] : process.env.WDIO_LOG_LEVEL as log.LogLevelDesc
 
         loggers[logName].setLevel(logLevel)
     })
 }
-const getLogLevelName = logName => logName.split(':').shift()
+const getLogLevelName = (logName: string): log.LogLevelDesc => logName.split(':').shift() as log.LogLevelDesc
```

Comment:
The return type isn't needed since it already knows it is going to be of the type log.LogLevelDesc